optuna/logging.py

```diff
@@ -82,6 +82,35 @@ def get_logger(name: str) -> logging.Logger:
 def get_verbosity() -> int:
     """Return the current level for the Optuna's root logger.
 
+    Example:
+
+        Get the default verbosity level:
+
+        .. testsetup::
+
+            def objective(trial):
+                x = trial.suggest_float("x", -100, 100)
+                y = trial.suggest_categorical("y", [-1, 0, 1])
+                return x ** 2 + y
+
+        .. testcode::
+
+            import optuna
+
+            # The default verbosity level of Optuna is `optuna.logging.INFO`.
+            print(optuna.logging.get_verbosity())
+            # 20
+            print(optuna.logging.INFO)
+            # 20
+
+            # There are logs of the INFO level.
+            study = optuna.create_study()
+            study.optimize(objective, n_trials=5)
+            # [I 2021-10-31 05:35:17,232] A new study created ...
+            # [I 2021-10-31 05:35:17,238] Trial 0 finished with value: ...
+            # [I 2021-10-31 05:35:17,245] Trial 1 finished with value: ...
+            # ...
+
```

Comment:
I'm not fully sure, but `testoutput` is required since the verbosity levels are shown in stdout.
https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-testoutput

```suggestion
            # [I 2021-10-31 05:35:17,245] Trial 1 finished with value: ...
            # ...

        .. testoutput::

           20
           20           
```

Reply:
Thank you for the fix.